includes/abstracts/abstract-batch.php

```diff
@@ -88,6 +88,13 @@
 	public $total_num_results;
 
 	/**
+	 * Difference in number of results between steps
+	 *
+	 * @var int
+	 */
+	public $results_changed = 0;
```

Comment:
Looking further down at https://github.com/reaktivstudios/locomotive/pull/96/files#diff-5eab949521aa84753b39a4ffceebf371R317 it looks like this isn't actually the difference it's just the number of changed results? If so I think the comment for this var might be misleading.